### PR TITLE
Bumped Up WP and PHP Minimums

### LINKED ISSUE #93: Bump WordPress and PHP minimums
Bump WP & PHP Minimums to the following
 
- [ ] WordPress to 5.7
- [ ] PHP to 7.4
- [ ] Remove any no-longer-needed conditional code for older version support.
- [ ] Take a look at adjusting our testing matrices, if applicable.
- [ ] Bump the minimums in plugin documentation and header fields.

### PR DESCRIPTION
**Pull request by zamanq**
### Description of the Change
Bumped up WP and PHP minimums. Also having some issues with the README.MD file formatting while saving so skipped that file to update the version numbers. 
Closes #93 

### Changelog Entry
Added - Bumped up WordPress and PHP Minimums

### Credits
Props @jeffpaul @zamanq 

### Checklist:
- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	],
 	"minimum-stability": "dev",
 	"require": {
-		"php": ">=5.4"
+		"php": ">=7.4"
```
Comment (cadic) [inline composer.json:18]: I wonder is 7.4 a real requirement within the codebase of this plugin? 7.0-7.3 still take more than 20% of setups according to [WordPress Stats](https://wordpress.org/about/stats/)

**Review by cadic — CHANGES_REQUESTED** [review]
@zamanq thanks for this update. I am concerned about using 7.4 here instead of 7.0 which looks a better option to me.

File: composer.json (hunk 1 above)
Comment (jeffpaul) [reply composer.json:18]: @cadic I imagine 7.4 was taken from the linked issue #93 which is what we want to bump our plugin minimums to at this point

Comment (jeffpaul) [reply composer.json:18]: @cadic that sound ok to you or should we stick with 7.0?

**Review by cadic — APPROVED** [review]

**Pull request merged**